b2b-sample/README.md

```diff
@@ -47,6 +47,8 @@ Now we need **two** applications to communicate with the **Guardio-Business-App*
 
 Create two applications named **Guardio-Business-App** and **Guardio-Admin-App** [This should be a management application].
 
+Share both applications with all the organizations by clicking the share button.
```

Comment:
There was no place mentioning about application sharing. Also the applications have to be shared at the moment of creation to view the organization_switch grant.